submissions/AsyaYeromina/Tiny-JS-world-OOP/index.js

```diff
@@ -0,0 +1,146 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: https://github.com/AsyaYeromina/a-tiny-JS-world
+  Web app: https://asyayeromina.github.io/a-tiny-JS-world/.
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+  constructor(inhabitantProperties) {
+    this.species = inhabitantProperties.species;
+    this.name = inhabitantProperties.name;
+    this.gender = inhabitantProperties.gender;
+    this.legs = inhabitantProperties.legs;
+    this.hands = inhabitantProperties.hands;
```

Comment:
Not all inhabitants have hands. Imagine how owehwhelmed becomes this class when we add birds, rhinos, fish etc
The beauty of OOP is that every class inherits shared props and adds what is unique to it and its children.
Revisit [the materials from the original project](https://github.com/OleksiyRudenko/a-tiny-JS-world#leveling-your-skills-up).

Let also every class handle its own properties. Note that base class shouldn't know and handle props introduced by any descendants.

Reply:
Ok, I've understood. I've added hands only for Sapiens class. 
I've also added tail only for Animals as an example of sharing props with all animals.